lua/nvim-tree/view.lua

```diff
@@ -202,12 +202,23 @@ function M.focus(winnr, open_if_closed)
   a.nvim_set_current_win(wnr)
 end
 
+local function handle_width()
```

Comment:
This function name is somewhat ambiguous. Maybe `get_width` would be more descriptive?